javascript/stalin-sort.js

```diff
@@ -1,11 +1,15 @@
-function stalinSort (array) {
-  const sorted = [array[0]];
-  for (let i = 1; i < array.length; ++i) {
-    if (array[i] >= array[i-1]) {
-      sorted.push(array[i])
+function stalinSort(array) {
+  if (!Array.isArray(array)) return [array];
```

Comment:
in this case i think is better to throw an error here

Reply:
Solved.